File: co.edu.uniandes.fixml.toCSharp/test/test3.xml/OrderType.cs

using System.Collections.Generic;
// Class for tag OrderType
class OrderType {


	List<LimitOrder> _LimitOrders = new List<LimitOrder>();
	
	// default constructor
	
	public OrderType ( ) {   
		;
		
		LimitOrder OrderType_LimitOrder= new LimitOrder("2");
		Price OrderType_LimitOrder_Price= new Price();
		OrderType_LimitOrder.AddPrice( OrderType_LimitOrder_Price );
		
		this.AddLimitOrder( OrderType_LimitOrder );
		
		
	}
	
	public OrderType (  ) {
	
	}
	
	public void AddLimitOrder( LimitOrder child ) {
		this._LimitOrders.Add( child );
	}
	
	
} 

File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test3/Java/OrderType.java
package object;

import java.util.List;
import java.util.ArrayList;

// Class for tag OrderType
public class OrderType {


  List<LimitOrder> LimitOrders = new ArrayList<LimitOrder>();



  public OrderType ( ) {   
  
;
  
LimitOrder OrderType_LimitOrder_0= new LimitOrder();
Price OrderType_LimitOrder_0_Price_0= new Price();
OrderType_LimitOrder_0.addPrice( OrderType_LimitOrder_0_Price_0 );
  		
this.addLimitOrder( OrderType_LimitOrder_0 );
  		
  }
      
  public void addLimitOrder( LimitOrder child ) {
    this.LimitOrders.add( child );
  }

} 